contributingGuides/CONTRIBUTING.md

```diff
@@ -32,9 +32,9 @@ This project and everyone participating in it is governed by the Expensify [Code
 At this time, we are not hiring contractors in Crimea, North Korea, Russia, Iran, Cuba, or Syria.
 
 ## Slack channels
-All contributors should be a member of a shared Slack channel called [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q) -- this channel is used to ask **general questions**, facilitate **discussions**, and make **feature requests**.
+We have a shared Slack channel called #expensify-open-source — this channel is used to ask general questions, facilitate discussions, and make feature requests.
```

Comment:
Did we mean to remove the hyper link here for #expensify-open-sourc? Previously it linked to the slack channel.

Reply:
Yes, it's intentional because the link doesn't benefit folks if they can't access.